scripts/setup.py

```diff
@@ -110,26 +110,56 @@ def build_cmake(self, ext):
 
 
```

Comment:
This file can be deleted. `source/python/setup.py.in` is used to generate `setup.py`.

Reply:
Yeah, I figured that out.